test/plugin.test.js

```diff
@@ -855,12 +855,14 @@ test('pluginTimeout', t => {
   })
   fastify.ready((err) => {
     t.ok(err)
-    t.equal(err.code, 'ERR_AVVIO_PLUGIN_TIMEOUT')
+    t.equal(err.message,
+      "fastify-plugin: Plugin did not start in time: 'function (app, opts, done) { -- // to no call done on purpose'. You may have forgotten to call 'done' function or to resolve a Promise")
```

Comment:
Could we add a test with a named function too?

Reply:
Sure, made a new one in cbb0e73